playkit/src/main/java/com/kaltura/playkit/PlayerEvent.java

```diff
@@ -113,10 +113,12 @@ public MetadataAvailable(List<PKMetadata> metadataList) {
     public static class SourceSelected extends PlayerEvent {
 
         public final String sourceUrl;
+        public final String sourceExtention;
 
-        public SourceSelected(String sourceUrl) {
+        public SourceSelected(String sourceUrl, String sourceExtention) {
```

Comment:
We should probably just send the selected MediaSource object. 
@orenme we send the raw url, right? I mean without playSessionId and friends.